Makefile

```diff
@@ -12,7 +12,7 @@ MIN_OPENSHIFT_VERSION ?= 4.12
 
 TEMPO_IMAGE ?= docker.io/grafana/tempo:$(TEMPO_VERSION)
 JAEGER_QUERY_IMAGE ?= docker.io/jaegertracing/jaeger-query:$(JAEGER_QUERY_VERSION)
-TEMPO_QUERY_IMAGE ?= docker.io/grafana/tempo-query:$(TEMPO_QUERY_VERSION)
+TEMPO_QUERY_IMAGE ?= ghcr.io/frzifus/tempo-query:$(TEMPO_QUERY_VERSION)
```

Comment:
I'm not sure if we should start linking to private repos, maybe we should wait until it's merged upstream or link to the `os-observability` org on GitHub?

Reply:
It actually got merged today. I will wait for the next nightly build and replace it :)